paddle/fluid/operators/checkpoint_load_op.cc

```diff
@@ -0,0 +1,214 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I see there's `load_op` and `load_combine_op` and corresponding saving ops, on python side, you can also use `fluid.io.save_persistables` to save all persistable variables.

In order to make `save_persistables` equal to save a checkpoint, make sure that the state variables are all "persistable" like step counters, learning rates, learning_rate moments etc.

So can you reuse those ops instead of writing some new one?

Reply:
```load_op``` and ```save_op``` are designed for ```LodTensor variable```,  But checkpoint will save variables not only ```LodTensor```， and ```checkpoint``` has some arguments particular.
At present, ```checkpoint load/save op``` and ```load/save op``` have no clear-cut distinction.

Reply:
I think it's better to reuse current operators maybe check the variable type will be fine.
So what the other variable types are saved in the checkpoint? "RAW" types and "feed" "fetch" may not need to be saved.